I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 73, in script
    show michael nervous right with dissolve
  File "game/script.rpy", line 73, in <module>
    show michael nervous right with dissolve
NameError: name 'diss' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 73, in script
    show michael nervous right with dissolve
  File "D:\renpy-8.0.3-sdk\renpy\ast.py", line 1651, in execute
    trans = renpy.python.py_eval(self.expr)
  File "D:\renpy-8.0.3-sdk\renpy\python.py", line 1092, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "D:\renpy-8.0.3-sdk\renpy\python.py", line 1085, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/script.rpy", line 73, in <module>
    show michael nervous right with dissolve
NameError: name 'diss' is not defined

Windows-10-10.0.19043 AMD64
Ren'Py 8.0.3.22090809
Dearth 1.0
Mon Sep 26 05:53:25 2022
